src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/SecretsAppSI.cs

```diff
@@ -36,7 +36,7 @@ public SecretsAppSI(IOptions<KeyVaultSettings> keyVaultSettings)
         public async Task<byte[]> GetCertificateAsync(string certificateId)
         {         
             using KeyVaultClient client = new KeyVaultClient(new KeyVaultClient.AuthenticationCallback(_azureServiceTokenProvider.KeyVaultTokenCallback));
-            CertificateBundle cert = await client.GetCertificateAsync(certificateId);
+            CertificateBundle cert = await client.GetCertificateAsync(_vaultUri, certificateId);
```

Comment:
certificateId should be the full uri including the path to the kv.. Parameter name should be changed to certificateName both here and in the interface.. I've updated this on a local branch and will be pushing out nugets for testing.